pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep.go

```diff
@@ -40,10 +41,7 @@ func newRemoveNatsOperatorStep() *removeNatsOperatorStep {
 }
 
 func defaultKubeClientProvider(context *service.ActionContext, logger *zap.SugaredLogger) (kubernetes.Client, error) {
-	kubeClient, err := kubernetes.NewKubernetesClient(context.Task.Kubeconfig, logger, &kubernetes.Config{
-		ProgressInterval: progressTrackerInterval,
-		ProgressTimeout:  progressTrackerTimeout,
-	})
+	kubeClient, err := kubernetes.NewKubernetesClient(context.Task.Kubeconfig, logger, nil)
```

Comment:
The initiation of this config is done inside NewKubernetesClient, if needed, it can also init from param. And these two constant values are from `pkg/reconciler/instances/eventing/preaction/common.go`, they are the same as our default value, that's why I removed the initiation from here.

We can of cause sync with the eventing team about this change, but anyway, this timeout and interval are not such critical variables.

Reply:
Ah I see, then we can remove their constants if not used anywhere else.

Regarding sync, you will have to get their review anyway to merge this PR since they are owners of `pkg/reconciler/instances/eventing`